general/GREY.md

```diff
@@ -0,0 +1,20 @@
+<!-- This file was generated by the script. Do not edit it, any changes will be lost! -->
+
+## GREY()
+
+
+
+Returns gray value which can be used in lcd functions
```

Comment:
man 2 time:
```
DESCRIPTION
       time() returns the time as the number of seconds since the Epoch, 1970-01-01 00:00:00 +0000 (UTC).
```
man stat:
```
DESCRIPTION
       These functions return information about a file. 
```

So both term are used interchangeably, but in this case I think returns is the right word.


```